packages/sakuli-commons/package.json

```
@@ -28,20 +28,20 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/common-tags": "^1.8.0",
-    "@types/jest": "^24.0.13",
+    "@types/jest": "^25.1.4",
     "@types/node": "^11.13.5",
     "@types/properties-reader": "^0.0.1",
-    "@types/yargs": "^12.0.12",
     "jest": "^24.7.1",
     "sneer": "^1.0.1",
-    "ts-jest": "^24.0.2",
+    "ts-jest": "^25.3.0",
     "typedoc": "^0.15.0",
     "typescript": "^3.4.3"
```

```suggestion
    "typescript": "^3.8.3"
```
I know, `^` makes the expressions the same but to ensure that we have the same version in place everywhere when we remove `^` finally.